client/my-sites/site-settings/form-general.jsx

```diff
@@ -260,12 +290,12 @@ const FormGeneral = React.createClass( {
 						name="blog_public"
 						value="1"
 						checked={ 1 === parseInt( this.state.blog_public, 10 ) }
-						onChange={ this.handleRadio }
+						onChange={ this.handleChange }
 						disabled={ this.state.fetchingSettings }
 						onClick={ this.onRecordEvent( 'Clicked Site Visibility Radio Button' ) } />
-					<span>{ this.translate( 'Public' ) }</span>
+					<span>{ translate( 'Public' ) }</span>
 					<FormSettingExplanation isIndented>
```

Comment:
If we strive to be consistent, I'd rather keep it non-indented because of how it is already in the Discussion tab in #10167 - see how the "These settings may be overriden..." text appears, and it's not indented. Anyway, let's also see what @MichaelArestad has to say about it.

> Can we ensure that all three types of inputs fix within 24px and that text labels are always exactly 24px (or whatever it is) from the left justification?

Sure, but I think that for Radio buttons it would create a huge white gap. Mini toggles are 24px wide, and have 21px of space to the right, which totals 45px. Radio buttons (together with white space) are equal to 21px. If we add another 24px to the right, the white gap would be too big IMO.